rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -79,5 +79,185 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="xml-renamed-components-00004">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-hdfs2" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-hdfs2` artifact has been renamed to `org.apache.camel:camel-hdfs`" effort="1" category-id="mandatory" >
+                    <message>`org.apache.camel:camel-hdfs2` artifact has been renamed to `org.apache.camel:camel-hdfs`. It’s corresponding component package was renamed from `org.apache.camel.component.hdfs2` to `org.apache.camel.component.hdfs`. The supported scheme is now `hdfs`.</message>
+                    <link title="Camel 3 - Renamed Components" href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_renamed_components" />
+                    <quickfix type="REPLACE" name="camel-hdfs2-replace">
+                        <replacement>camel-hdfs2</replacement>
+                        <search>camel-hdfs</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-renamed-components-00005">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-hdfs" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-hdfs` artifact has been replaced with the old `org.apache.camel:camel-hdfs2` artifact" effort="1" category-id="mandatory" >
```

Comment:
For consistency with the previous hints' titles
Also the category might change to `potential` since it's a 3rd party library (Hadoop) change we don't know enough about
```suggestion
                <hint title="`org.apache.camel:camel-hdfs` artifact has been replaced" effort="1" category-id="potential" >
```